tick/array/tests/serialize_test.py

```diff
@@ -60,6 +60,33 @@ def test_serialize_sparse_2d_array(self):
         # python 3.5 has show to required this - investigate typemappers
         gc.collect()
 
+    def test_serialize_column_major_2d_array(self):
+        """...Test serialization of 2d dense array is done as expected
+        """
+        x = np.array([[1, 2.1, 5], [0, 4, 1]]).astype(self.dtype)
+        array = np.asfortranarray(x).reshape(3, 2)
+        serialize_array(array, self.array_file)
+
+        serialized_array = load_array(self.array_file, array_dim=2,
+                                      dtype=self.dtype)
+        np.testing.assert_array_almost_equal(array, serialized_array)
```

Comment:
What happens if you add
`np.testing.assert_array_almost_equal(x, serialized_array)` to compare it to its row major version ?